tools-local/unix/cmake/get-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Gets the version of CMake at the specified path.
+
+usage()
+{
+    echo "Usage: $0 cmake ToolName --ToolPath <path>"
```

Comment:
>ToolPath [](start = 37, length = 8)

Would --tool-path be more unix friendly?  You wouldn't have to make it case sensitive as well.